Modules/Tokens/test/Tokens.API.Tests.Integration/StepDefinitions/TokensApiStepDefinitions.cs

```diff
@@ -0,0 +1,263 @@
+                break;
+            default:
+                requestConfiguration.ExpiresAt = DateTime.Now.ToString("dd-MM-yyyy");
+                break;
+        }
+
+        _tokenResponse = await _tokensApi.CreateToken(requestConfiguration);
+    }
+
+    [When(@"a POST request is sent to the Tokens endpoint with no request content")]
+    public async Task WhenAPOSTRequestIsSentToTheTokensEndpointWithNoRequestContent()
+    {
+        _requestConfiguration.Content = null;
+        _requestConfiguration.ExpiresAt = null;
+        _tokenResponse = await _tokensApi.CreateToken(_requestConfiguration);
+    }
+
+    [When(@"a GET request is sent to the Tokens/{id} endpoint with ""?(.*?)""?")]
+    public async Task WhenAGETRequestIsSentToTheTokensIdEndpointWith(string id)
+    {
+        switch (id)
+        {
+            case "t.Id":
+                id = _tokenId!;
+                break;
+            case "p.Id":
+                id = _peerTokenId!;
+                break;
+            case "a valid Id":
+                id = "TOKjVPS6h1082AuBVBaR";
+                break;
+        }
+        _tokenResponse = await _tokensApi.GetTokenById(_requestConfiguration, id);
+    }
+
+    [When(@"a POST request is sent to the Tokens endpoint with '([^']*)', '([^']*)'")]
+    public async Task WhenAPOSTRequestIsSentToTheTokensEndpointWith(string content, string expiresAt)
+    {
+        var requestConfiguration = new RequestConfiguration
+        {
+            Content = content,
+            ExpiresAt = expiresAt
+        };
+
+        requestConfiguration.SupplementWith(_requestConfiguration);
+
+        var httpResponse = await _tokensApi.CreateToken(requestConfiguration);
+        _tokenResponse = httpResponse;
+    }
+
+    [Then(@"the response does not contain any peer Tokens")]
+    public void ThenTheResponseDoesNotContainAnyPeerTokens()
+    {
+        var ownToken = _createdTokens.First();
+
+        _responseTokens.Should().NotBeEmpty();
+
+        var ownTokenFromResponse = _responseTokens.FirstOrDefault(x => x.Id == ownToken.Id);
+        ownTokenFromResponse.Should().NotBeNull();
+
+        _responseTokens.All(token => token.CreatedBy == ownTokenFromResponse!.CreatedBy).Should().BeTrue();
+    }
+
+    [Then(@"the response content includes an error with the error code ""([^""]+)""")]
+    public void ThenTheResponseContentIncludesAnErrorWithTheErrorCode(string errorCode)
+    {
+        _tokenResponse.Data!.Error.Should().NotBeNull();
+
+        _tokenResponse.Data!.Error!.Code.Should().Be(errorCode);
+    }
+
+    [Then(@"the response contains a Token")]
+    public void ThenTheResponseContainsAToken()
+    {
+        _tokenResponse.Should().NotBeNull();
+
+        AssertStatusCodeIsSuccess();
+
+        AssertResponseContentTypeIsJson();
+
+        AssertResponseContentCompliesWithSchema();
+
+        if (_tokenResponse.HttpMethod == Method.Get.ToString())
+        {
+            AssertExpirationDateIsInFuture();
```

Comment:
I'm assuming you're asking about the 
````csharp
if (_tokenResponse.HttpMethod == Method.Get.ToString())
````

If so, please check my answer to a previous comment: https://github.com/nmshd/backbone/pull/61#discussion_r1157393643

Reply:
No, I'm actually referring to the highlighted line (`AssertExpirationDateIsInFuture`). 
I assume you copied it from the Challenges tests. But for the Tokens, the expiration date is passed as a parameter during creation. And if it is in the past, the POST endpoint returns a 400.
So this assertion will never fail and can therefore be removed.

Reply:
To be precise: it CAN fail if too much time has passed since the creation of the Token and the Get. But since all of your Tokens expire "tomorrow", this will never be the case.